server/router/test.js

```diff
@@ -1,3 +1,19 @@
-test('this is sample test', () => {
-   console.log("I am running lots of tests!"); 
+const app = require("../app.js")
+const supertest = require("supertest")
+const requestWithSupertest = supertest(app);
+
+describe('Testing Routes', function () {
+
+  it('responds to /', async function() {
+    const res = await requestWithSupertest.get('/');
+    expect(res.status).toEqual(200);
+    expect(res.text).toEqual("I am running! I'm gonna talk to a database!");
+  });
+
+  it('responds to /api/question1', async function() {
+    const res = await requestWithSupertest.get('/api/question1');
+    expect(res.status).toEqual(200);
+    expect(res.body).toHaveProperty('salaries');

```

Comment:
in addition to having these properties, can we add a check to see if the array is not empty /contains proper data?

Reply:
right now it's just checking to see if the body contain these properties, but they may not contain proper data

Reply:
Done